src/store/traction/pacbio/plates/actions.js

```diff
@@ -1,16 +1,20 @@
 import handlePromise from '@/api/PromiseHelper'
 
-const setPlates = async ({ commit, getters }) => {
+const setPlates = async ({ commit, getters }, filter) => {
   let request = getters.getPlates
-  let promise = request.get({ include: 'wells.materials' })
+  let promise = request.get({
+    include: 'wells.materials',
+    filter: filter,
+  })
   let response = await handlePromise(promise)
 
   if (response.successful && !response.empty) {
     let plates = response.deserialize.plates
```

Comment:
I think at some point we can look at removing the usage of deserialize as I think it can cause all sorts of problems with recursion.

Reply:
Yep agreed